includes/emails/class-email-summary.php

```diff
@@ -0,0 +1,154 @@
+		}
+
+		return self::$instance;
+	}
+
+	private function hooks() {
+		add_filter( 'edd_settings_emails', array( $this, 'email_summary_settings' ) );
+		add_action( 'edd_trigger_email_summary', array( $this, 'trigger_email_summary' ) );
+		add_action( 'edd_email_summary_preview', array( $this, 'email_summary_preview' ) );
+	}
+
+	/**
+	 * Hooks into settings screen and adds additional
+	 * buttons and informations about Email Summary.
+	 */
+	public function email_summary_settings( $settings ) {
+		$email_summary_trigger_url = wp_nonce_url(
+			edd_get_admin_url(
+				array(
+					'page'       => 'edd-settings',
+					'tab'        => 'emails',
+					'section'    => 'email_summaries',
+					'edd_action' => 'trigger_email_summary',
+				),
+			),
+			'edd_trigger_email_summary'
+		);
+
+		$email_summary_preview_url = wp_nonce_url(
+			edd_get_admin_url(
+				array(
+					'edd_action' => 'email_summary_preview',
+				),
+			),
+			'edd_email_summary_preview'
+		);
+
+		$email_summary_schedule      = wp_next_scheduled( EDD_Email_Summary_Cron::CRON_EVENT_NAME );
+		$email_summary_schedule_text = '<span><span class="dashicons dashicons-warning"></span> ' . esc_html( __( 'Not scheduled!', 'edd-dev-tools' ) ) . '</span>';
+
+		if ( $email_summary_schedule ) {
+			$email_summary_schedule_date = \Carbon\Carbon::createFromTimestamp( $email_summary_schedule );
+			$email_summary_schedule_text = '<code>' . $email_summary_schedule_date->format( 'Y-m-d H:i:s' ) . '</code>';
+		}
+
+		$additional_settings = array(
+			array(
+				'id'   => 'email_summary_cron_event_name',
+				'name' => __( 'Cron Event Name', 'edd-dev-tools' ),
+				'desc' => '<code>' . EDD_Email_Summary_Cron::CRON_EVENT_NAME . '</code>',
+				'type' => 'descriptive_text',
+			),
+			array(
+				'id'   => 'email_summary_schedule',
+				'name' => __( 'Next Schedule', 'edd-dev-tools' ),
+				'desc' => $email_summary_schedule_text,
+				'type' => 'descriptive_text',
+			),
+			array(
+				'id'   => 'email_summary_send_button',
+				'name' => __( 'Trigger Email Summary', 'edd-dev-tools' ),
+				'desc' => '<a href="' . esc_url( $email_summary_trigger_url ) . '" class="button">'. esc_html( __( 'Send Email', 'easy-digital-downloads' ) ) . '</a>',
+				'type' => 'descriptive_text',
+			),
+			array(
+				'id'   => 'email_summary_preview',
+				'name' => __( 'Preview Email Summary', 'edd-dev-tools' ),
+				'desc' => '<a target="_blank" href="' . esc_url( $email_summary_preview_url ) . '" class="button">' . esc_html( __( 'Preview Email', 'easy-digital-downloads' ) ) . '</a>',
+				'type' => 'descriptive_text',
+			),
+		);
+
+		foreach ( $additional_settings as $setting ) {
+			$settings['email_summaries'][ $setting['id'] ] = $setting;
+		}
+
+		return $settings;
+	}
+
+	/**
+	 * Send Email Summary.
+	 */
+	public function trigger_email_summary() {
+		if ( ! isset( $_REQUEST['_wpnonce'] ) || ! wp_verify_nonce( $_REQUEST['_wpnonce'], 'edd_email_summary_preview' ) ) {
```

Comment:
```suggestion
		if ( ! isset( $_REQUEST['_wpnonce'] ) || ! wp_verify_nonce( $_REQUEST['_wpnonce'], 'edd_trigger_email_summary' ) ) {
```
This was working for me before but today it's not ... is it checking for the wrong nonce?